.github/workflows/nextjs-bundle-analysis.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           body="${body//'%'/'%25'}"
           body="${body//$'\n'/'%0A'}"
           body="${body//$'\r'/'%0D'}"
-          echo ::set-output name=body::$body
+          echo "body=$body" >> "$GITHUB_OUTPUT"
```

Comment:
I had tried this in the past and it caused major output issues on the PR

Reply:
Let’s not include on this PR and instead make a separate PR that targets only this change

Reply:
sure keith